vercel/index.html

```diff
@@ -4,29 +4,14 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>HTML5 Test Page</title>
-    <link rel="stylesheet" href="css/normalize.css" />
-    <link id="sakura-css" rel="stylesheet" href="css/sakura.css" />
+    <style id="sakura-css"></style>
```

Comment:
On the final rendered pages, `style.css` is being added below sakura styles. This leads to `normalize.css` overriding some of sakura. Can we put sakura below index.css
![image](https://user-images.githubusercontent.com/4194289/182534361-21ce6d02-abe7-47f7-b4cd-ce92a4b7f53a.png)